internal/blockchain/ethereum/ethereum.go

```diff
@@ -32,8 +32,9 @@ import (
 )
 
 type Account struct {
-	Address    string `json:"address"`
-	PrivateKey string `json:"privateKey"`
+	Address      string `json:"address"`
+	PrivateKey   string `json:"privateKey"`
+	PtmPublicKey string `json:"ptmPublicKey"`
```

Comment:
```suggestion
	PtmPublicKey string `json:"ptmPublicKey"` // Private Transaction Key used for Tessera
```

Reply:
I need to circle back on this one based on the other conversation around the programming model for this

Reply:
Got it, Tessera public keys are heavily used for private networks, so it would be good if the data point is easily accessible